spec/scenarios/posts_spec.rb

```diff
@@ -0,0 +1,94 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts index page', type: :feature do
+  before :each do
+    @user = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
+    @user2 = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
+    @first_post = Post.create(author: @user, title: 'Hello', text: 'This is my post')
+    5.times do |_i|
+      Comment.create(post: @first_post, author: @user2, text: 'Hi Lilly!')
+    end
+    visit user_posts_path(@user)
+  end
+
+  scenario 'displays User profile Picture' do
+    find('img[alt="Lilly"]')
+  end
+
+  scenario 'display the User Name' do
+    expect(page).to have_content(@user.name)
+  end
```

Comment:
- [ ] Kindly,  it will be great to add instructions on how to run the integration `tests`. Actually, I'm getting this error message 👇🏻

![02NOW](https://user-images.githubusercontent.com/79058364/181286972-640e92e2-eb75-49e8-b69f-50eac8c06ef2.png)

Reply:
ok noted